source/docs/kubernetes.md

```diff
@@ -1,3 +1,89 @@
-# Managing Containers
+# Kubernetes in Atomic Host
 
-[Kubernetes](http://kubernetes.io/) is the command-line interface to managing containers. You can [find more](https://kubernetes.io/docs/user-guide/) information and detailed documentation about Kubernetes from the various project pages.
+Atomic Hosts rely primarily on Kubernetes for automating deployment, scaling, and operations of application containers across clusters of hosts.
+
+Initially, the Atomic Hosts from Fedora and CentOS shipped with Kubernetes included in the image by default, but the projects have been moving toward containerized Kubernetes deployments, with the goal of making it easier to deploy different versions. 
+
+Below is an overview of the state of Kubernetes in Fedora Atomic and CentOS Atomic, and some information on other installation options. 
+
+## Fedora Atomic Host
+
+As of Fedora 25, Kubernetes, as well as Etcd and Flannel, [is included](https://pagure.io/fedora-atomic/blob/f25/f/fedora-atomic-docker-host.json#_113) in the default image.
```

Comment:
Most of the time, it's better to link to a specific hash so that the lineno doesn't get out of date. E.g. https://pagure.io/fedora-atomic/blob/9ca7cc0c806014576a6413ce5aac3db72c421f5d/f/fedora-atomic-docker-host.json#_113.